### PR TITLE
 #306 - Add renderOnFetch option to DataBehavior

### PR DESCRIPTION
**Pull request by mandragorn**
* Updated tests & docs to match.
* renderOnFetch = true will cause the view to render when a fetched
  event is received.
* renderOnFetch defaults to false.

### CODE REVIEW

**Comment by mandragorn** [comment]
@kentmw this is ready for review.

**Comment by mandragorn** [comment]
now its more ready for review...

**Comment by fyliu24** [comment]
+1 looks good to me

File: docs/modules/behaviors/DATA_BEHAVIOR.md [hunk 1]
```diff
@@ -807,9 +808,4 @@ successEventPayload = {
   * "behaviors.behaviorAlias:[event name]" - arbitrary event triggered by another behavior on this view.
   * "behaviors.behaviorAlias.[property name]:[event name]" - arbitrary event triggered by another behavior's property on this view.
   * "behaviors.behaviorAlias.data:[event name]" - arbitrary event triggered by another behavior's data property on this view (specific example of `behaviorAlias.[property name]:[event name]`).
-  * { '[event name]': < object (or function returning an object) that the event is triggered on > } - arbitrary "event" triggered on the supplied object.
-
-
-
-
-
+  * { '[event name]': < object (or function returning an object) that the event is triggered on > } - arbitrary "event" triggered on the supplied object.
```
Comment (kentmw) [inline docs/modules/behaviors/DATA_BEHAVIOR.md]: I feel like this comment could get a little love. I read it a couple times and still wasn't sure what its purpose was. Is it triggering [event name] on the object, or listening to [event name] on the object, and then doing what when it is triggered?

Comment (mandragorn) [reply docs/modules/behaviors/DATA_BEHAVIOR.md]: see #341

**Pull request merged**